src/types.rs

```diff
@@ -1242,6 +1242,16 @@ impl<T: FromRedisValue> FromRedisValue for Option<T> {
     }
 }
 
+#[cfg(feature = "bytes-comp")]
```

Comment:
```suggestion
#[cfg(feature = "bytes")]
```
And remove the `bytes-comp` feature, it is not necessary.

Reply:
Done.